__tests__/fixtures/react-native/fabricRefRewrites/reactDevelopment/output.js

```diff
@@ -714,7 +714,8 @@ if (process.env.NODE_ENV !== 'production') {
         if (
           type.$$typeof &&
           (type.$$typeof.toString() === 'Symbol(react.forward_ref)' ||
-            type.$$typeof.toString() === 'Symbol(react.element)')
+            type.$$typeof.toString() === 'Symbol(react.element)' ||
+            type.$$typeof.toString() === 'Symbol(react.transitional.element)')
```

Comment:
React element name `Symbol(react.element)` is now [legacy](https://github.com/facebook/react/blob/b10cb4c01ec1ae41b67422239d919f261fefa7d1/packages/react-devtools-shared/src/backend/shared/ReactSymbols.js#L28)